seaworthy/pytest/fixtures.py

```diff
@@ -14,7 +14,7 @@ def docker_helper_fixture(name='docker_helper', scope='module', **kwargs):
     Create a fixture for :class:`~seaworthy.DockerHelper`.
 
     This can be used to create a fixture with a different name to the default.
-    It can also be used to override the scope of the default fixture:::
+    It can also be used to override the scope of the default fixture::
```

Comment:
`T_PAAMAYIM_NEKUDOTAYIM`! :trollface:

Reply:
😱 wow

Reply:
I tried to find an appropriate translation of "thrice colon" ("pa'amayim nekudotayim" is the transliteration of "פַּעֲמַיִם נְקוּדָתַיִם" which translates to "twice colon") to put in there, but my Hebrew isn't good enough to keep the same form without turning it into "three colon" or "colon three times" and neither of those are sufficiently recognisable for the troll to work.